x-pack/test/security_solution_ftr/services/detections/index.ts

```diff
@@ -128,4 +170,62 @@ export class DetectionsTestService extends FtrService {
       }
     );
   }
+
+  /**
+   * Loads alerts for Endpoint directly into the internal index that the Endpoint Rule
+   * would have written them to for a given endpoint
+   * @param endpointAgentId
+   * @param count
+   */
+  async loadEndpointRuleAlerts(
+    endpointAgentId: string,
+    count: number = 2
+  ): Promise<IndexedEndpointRuleAlerts> {
+    this.log.info(`Loading ${count} endpoint rule alerts`);
+
+    await this.ensureEndpointRuleAlertsIndexExists();
+
+    const alertsGenerator = new EndpointRuleAlertGenerator();
+    const esClient = this.esClient;
+    const indexedAlerts: estypes.IndexResponse[] = [];
+
+    for (let n = 0; n < count; n++) {
+      const alert = alertsGenerator.generate({ agent: { id: endpointAgentId } });
+      const indexedAlert = await esClient.index({
+        index: `${DEFAULT_ALERTS_INDEX}-default`,
+        refresh: 'wait_for',
+        body: alert,
+      });
+
+      indexedAlerts.push(indexedAlert);
+    }
+
+    this.log.info(`Endpoint rule alerts created:`, indexedAlerts);
+
+    return {
+      alerts: indexedAlerts,
+      cleanup: async (): Promise<void> => {
+        if (indexedAlerts.length) {
+          this.log.info(`cleaning up loaded endpoint rule alerts`);
```

Comment:
This doesn't need template literals unless maybe you forgot to log some variables here.